apps/email-otp-authentication-portal/src/main/webapp/js/scripts.js

```diff
@@ -1,43 +0,0 @@
-/**
- * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
- *
- * WSO2 Inc. licenses this file to you under the Apache License,
- * Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied. See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-$(document).ready(function() {
-    $("#authorizeLink").click(function() {
-        $("#loginForm").show("slow");
-    });
-    $("#loginBtn").click(function() {
-        var error = "";
-        if ($("#oauth_user_name").val() == "") {
```

Comment:
How are we handling these validations in the new approach? since the script is removed...

Reply:
I didn't find any of these IDs in the webapp. Seems some legacy script.